custom/icds_reports/templates/icds_reports/mobile_login.html

```diff
@@ -1,3 +1,14 @@
 {% extends "icds/login.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
 {% block title %}{% trans "Login to ICDS CAS Mobile Dashboard" %}{% endblock title %}
+{% block page_content %}
+{{ block.super }}
+{% registerurl 'cas_mobile_dashboard' domain %}
```

Comment:
I think this can be removed; I don't see it referenced in js.

Reply:
Oops, I think I got my branches crossed. I fixed this in a rebase on https://github.com/dimagi/commcare-hq/pull/25950 but i didn't realize it was also on this branch...

Reply:
Actually nevermind I didn't actually fix it there either. Will fix now.